pkg/semtech-loramac/include/semtech_loramac.h

```diff
@@ -110,6 +110,7 @@ typedef struct {
 typedef struct {
     mutex_t lock;                                /**< loramac access lock */
     uint8_t caller_pid;                          /**< pid of caller thread */
```

Comment:
Maybe it makes more sense if this is `tx_pid` if I understand right this thread is in charge of join procedure and sending packets, so tx. Would be more inline with rx_pid (receiver thread)